.github/labeler.yml

```diff
@@ -5,6 +5,11 @@ CI/run-audit-deps:
 - "**/Cargo.toml"
 - "**/Cargo.lock"
 
+CI/run-network-audit:
+- browser/net/*
+- chromium_src/**/*features.cc
+- patches/chrome-VERSION.patch
```

Comment:
Looks good but it's missing the globs for the test suite-related changes:
```
- browser/net/brave_network_audit_browsertest.cc
- browser/net/brave_network_audit_allowed_lists.h
- ```

Reply:
Should be covered by `browser/net/*` above, no?

Reply:
ah, right. I was thinking of the tests without realizing are part of the same subdir. Forget it then :innocent: